new/LIT7094PrayerDevils.xml

```diff
@@ -70,7 +71,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <div xml:lang="gez" type="edition">
                 <note>The following incipit is taken from <ref type="mss" corresp="BLorient3663"/> as quoted in
                     <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">120</citedRange></bibl>.</note>
-                <div type="textpart" subtype="incipit">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
```

Comment:
I have seen, that xml:lang="gez" is already used in line 70 (new71). So it might be not necessary again in line 73 (new 74).